Tools/nuget/make_zip.py

```diff
@@ -1,4 +1,4 @@
-#! /usr/bin/python3
+#! /usr/bin/tauthon3
```

Comment:
Agreed. Looks like a bug. I know for sure brew doesn't pull python3 as dependency for python2.